apps/web/src/middlewares/errorHandler.ts

```diff
@@ -1,11 +1,16 @@
 import { LatitudeError } from '@latitude-data/core/lib/errors'
+import env from '$/env'
 import { NextRequest, NextResponse } from 'next/server'
 
 export function errorHandler(handler: any) {
   return async (req: NextRequest, res: NextResponse) => {
     try {
       return await handler(req, res)
     } catch (error) {
+      if (env.NODE_ENV === 'development') {
+        console.error(error)
+      }
```

Comment:
use `debug`. This way don't need to check env

Reply:
pues tiene razon